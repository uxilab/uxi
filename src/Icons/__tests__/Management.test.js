import React from 'react';
import Management from '../Management';

describe('Icon : <Management />', () => {
  it('match snapshot', () => {
    expect(shallow(<Management />)).toMatchSnapshot();
  });
});

