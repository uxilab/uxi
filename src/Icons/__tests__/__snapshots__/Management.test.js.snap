// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Management /> match snapshot 1`] = `
<SvgIcon
  color="currentColor"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 14.37 20.21"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M6.64,8.17h-1V5.56a.8.8,0,0,0-.8-.79A1.85,1.85,0,0,1,2.94,2.92V0H4V2.92a.79.79,0,0,0,.79.79A1.85,1.85,0,0,1,6.64,5.56Z"
        />
        <path
          d="M8.79,8.17H7.73V5.56A1.85,1.85,0,0,1,9.58,3.71a.79.79,0,0,0,.8-.79V0h1V2.92A1.85,1.85,0,0,1,9.58,4.77a.79.79,0,0,0-.79.79Z"
        />
        <path
          d="M7,5.58h.34v0a2.24,2.24,0,0,1,.46-1.34H6.55A2.18,2.18,0,0,1,7,5.56Z"
        />
        <path
          d="M11,4.64a2.19,2.19,0,0,1-1.39.5.42.42,0,0,0-.42.42v.15C11.57,6,13,6.84,13,7.33S10.79,9.08,7.19,9.08,1.37,8,1.37,7.33,2.8,6,5.21,5.71V5.56a.42.42,0,0,0-.42-.42,2.19,2.19,0,0,1-1.39-.5C1.44,5.13,0,6,0,7.33v9.75c0,2.05,3.62,3.13,7.19,3.13s7.18-1.08,7.18-3.13V7.33C14.37,6,12.93,5.13,11,4.64Zm2,12.44c0,.62-2.21,1.76-5.81,1.76S1.37,17.7,1.37,17.08V15.73l.19.1A12.7,12.7,0,0,0,7.19,17a12.73,12.73,0,0,0,5.63-1.13l.18-.1Zm0-3.25c0,.62-2.21,1.75-5.81,1.75s-5.82-1.13-5.82-1.75V12.48l.19.1A12.71,12.71,0,0,0,7.19,13.7a12.74,12.74,0,0,0,5.63-1.12l.18-.1Zm0-3.25c0,.62-2.21,1.75-5.81,1.75S1.37,11.2,1.37,10.58V9.23l.19.1a12.71,12.71,0,0,0,5.63,1.12,12.74,12.74,0,0,0,5.63-1.12l.18-.1Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
